test/ui-test/testSuites/suite_statusLogin/shared/steps/steps.py

```diff
@@ -14,10 +14,15 @@
 # *          with a pattern which is matched against the steps being executed.
 # *****************************************************************************
 from data.StatusAccount import StatusAccount 
-from processes.StatusLoginProcess import StatusLoginProcess    
+from processes.StatusLoginProcess import StatusLoginProcess
+from screens.StatusWelcomeScreen import StatusWelcomeScreen 
+from screens.StatusChatScreen import StatusChatScreen 
+
+welcomeScreen = StatusWelcomeScreen()  
```

Comment:
It should be better to create the object in the `Given` when you are doing the preconditions stuff and store the retreived object in the `context.userData`. Something like this:

```suggestion
context.userData['welcomeScreen'] = StatusWelcomeScreen() 
```

Reply:
I rather keep the context to storing actual data  (static) that need to keep state, and not instances of an object. 

If I am storing an instance, I rather just remove the class and use static functions.

If I have another step definition file and someone calls the welcome screen from the context, they will be using the same instance. I don't think I like that.

What do you think?

Reply:
Well.. I think you must ensure the context you are using is the right one you want to use. As said in a previous comment, `context.userData` propose is exactly to pass / share data between steps rather than having global properties, that can make the code not as clean and clear. Instead of using "static" data, maybe we can ensure the initialization in `hooks`.